packages/hash/api/src/graphql/resolvers/entity/linkGroups.ts

```diff
@@ -90,14 +101,18 @@ const addEntityAggregationResultOutgoingLinks =
   async (entity: Entity) => {
     const aggregations = await entity.getAggregations(client);
 
-    const allAggregationResults = (
-      await Promise.all(
-        aggregations.map((aggregation) => aggregation.getResults(client)),
-      )
-    )
+    const aggregationResults = await Promise.all(
+      aggregations.map((aggregation) => aggregation.getResults(client)),
+    );
+
+    const allAggregationResults = aggregationResults
       .flat()
       // filter duplicate resulting entities
-      .filter((result, i, all) => all.findIndex(result.isEquivalentTo) === i);
+      .filter((result, i, all) => {
+        return (
+          all.findIndex((entity) => areEntitiesEquivalent(entity, result)) === i
```

Comment:
This change fixes a bug with the existing implementation of `result.isEquivalentTo`, where `this` is appearing as `undefined`, causing the code to error out.